hoot-services/src/main/java/hoot/services/job/JobRunnable.java

```diff
@@ -93,6 +93,10 @@ private void processJob() {
             }
 
             jobStatusManager.setCompleted(job.getJobId(), "FULLY PROCESSED");
+
+            if(job.getJobType().equals(JobType.UPLOAD_CHANGESET)){
+                DbUtils.setStale(job.getParentId());
+            }
```

Comment:
should we just make this an internal command that is part of upload changeset?

Reply:
I retract this comment based on your observation that the line above this would simply overwrite the stale status_detail with FULLY PROCESSED.